### PR TITLE
show mentor and students

### PR DESCRIPTION
**Pull request by g4s8**
#448 - show mentor and students on profile page

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#577](https://github.com/zerocracy/farm/issues/577) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#577](https://github.com/zerocracy/farm/issues/577) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/471705172). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by carlosmiranda** [comment]
@yegor256 looks good to merge

File: src/main/xsl/profile.xsl [hunk 1]
```diff
@@ -300,4 +302,37 @@ SOFTWARE.
       <xsl:text>.</xsl:text>
     </p>
   </xsl:template>
+  <xsl:template match="mentor">
+    <p>
+      <xsl:text>Your mentor: </xsl:text>
+      <xsl:element name="a">
```
Comment (yegor256) [inline src/main/xsl/profile.xsl]: @g4s8 you can do it easily:

```xml
<a href="/u/{.}">
  <xsl:value-of select="text()"/>
</a>
```

File: src/main/xsl/profile.xsl [hunk 2]
```diff
@@ -300,4 +302,37 @@ SOFTWARE.
...
+      <xsl:text>You don't have students yet.</xsl:text>
+    </p>
+  </xsl:template>
+  <xsl:template match="students[student]">
+    <p>
+      <xsl:text>Your students: </xsl:text>
+      <xsl:element name="ul">
+        <xsl:for-each select="student">
+          <li>
+            <xsl:element name="a">
```
Comment (yegor256) [inline src/main/xsl/profile.xsl]: @g4s8 same here

**Comment by yegor256** [comment]
@g4s8 see above

**Comment by yegor256** [comment]
@rultor merge

**Comment by g4s8** [comment]
@yegor256 resolved conflict, please restart merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was finished, quality was "acceptable": +15 points just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), total is [+1440](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+10369](https://www.0crat.com/u/yegor256)

**Pull request merged**